packages/vendure-plugin-invoices/src/strategies/load-data-fn.ts

```diff
@@ -71,15 +72,15 @@ export const defaultLoadDataFn: LoadDataFn = async (
   });
   if (!shouldGenerateCreditInvoice) {
     // Normal debit invoice
-    return {
+    return Promise.resolve({
```

Comment:
This is probably caused because you removed the `async` above right? This makes it not return a promise anymore.

However, the cleaner way of fixing this is to change the method signature of the LoadDataFn interface to also allow returning of a non promise: `Promise<InvoiceData> | InvoiceData`. This way we don't have to do Promise.resolve for a function that isnt doing anything async